### PR TITLE
Add PHP5.6 && PHP7.1

### PR DESCRIPTION
**Pull request by LoreleiAurora**
This makes Varying-Vagrant-Vagrants/vvv#1055 useful.

### CODE REVIEW

File: php71/provision.sh [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+    apt-get autoremove -y
+
+    # Clean up apt caches
+    apt-get clean
+  fi
+}
+
+configure() {
+  # Copy nginx configuration from local
+  cp "${DIR}/php7.1-upstream.conf" "/etc/nginx/upstreams/php56.conf"
```
Comment (jeremyfelt) [inline php71/provision.sh]: Looks like this should be copying to `php71.conf` instead of `php56.conf`

File: php56/provision.sh [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+  php5.6-soap
+  php5.6-ssh2
+  php5.6-xdebug
+  php5.6-xml
+  php5.6-zip
+)
+
+### FUNCTIONS
+
+network_detection() {
```
Comment (jeremyfelt) [inline php56/provision.sh]: I wonder if we can add VVV core's `network_detection()` to the box in a way that makes it reusable for other provisioning scripts. :thinking:

File: php71/provision.sh (hunk 1 above)
Comment (LoreleiAurora) [inline php71/provision.sh]: Good eye, i have fixed it now, this was the only line left on my work tree.

File: php56/provision.sh (hunk 2 above)
Comment (LoreleiAurora) [inline php56/provision.sh]: I have an idea how we can, but i would rather get the feature in then do that in a separate pr when i solve it in VVV core.

Comment (jeremyfelt) [inline php56/provision.sh]: Yup, that makes sense. :+1:

**Comment by jeremyfelt** [comment]
Ok, all good here. I haven't been able to test thoroughly, but I think it's better to have this in early. I'll be doing more hands on testing this weekend to back @LoreleiAurora up. :+1:
